Podfile.lock

```diff
@@ -0,0 +1,18 @@
+PODS:
+  - MSAL (1.1.13):
```

Comment:
Can this be updated to latest MSAL? Right now when I run pod install it seems to use MSAL 1.1.13